README.rst

```diff
@@ -824,15 +824,19 @@ The configuration of this feature is stored in the configuration variable *trace
 .. code-block:: python
 
     traceability_checklist = {
-        'private_token': config('PRIVATE_TOKEN'),
-        'api_host_name': config('API_HOST_NAME'),
-        'project_id': config('PROJECT_ID'),
-        'merge_request_id': config('MERGE_REQUEST_ID'),
-        'attribute_name': config('TARGET_ATTRIBUTE_NAME'),
-        'attribute_to_str': config('TARGET_ATTRIBUTE_TO_STRING'),
-        'attribute_values': config('TARGET_ATTRIBUTE_VALUES'),
+        'private_token': config('PRIVATE_TOKEN', ''),
```

Comment:
we want this as: `'private_token': 'enteryourprivatetokenhere'`

Reply:
4aa906a5c02bef51c80f513121115eb9ce4cb540